gym/core.py

```diff
@@ -51,6 +51,7 @@ def __new__(cls, *args, **kwargs):
         env = super(Env, cls).__new__(cls)
         env._env_closer_id = env_closer.register(env)
         env._closed = False
+        env._spec = None
```

Comment:
Seems a bit hacky. Isn't it less likely to cause confusion if it's a property in both places?